### DISCUSSION TITLE
Guru Meditation Error: Core  1 panic'ed (LoadProhibited)

### DISCUSSION THREAD

**Discussion by Finnn-glitch** [schreibfaul1/ESP32-audioI2S#584] [Q&A]
Hey so i copy pasted the example code but my esp32 just gives me the error LoadProhibited.
Anyone know what i did wrong?
![image](https://github.com/schreibfaul1/ESP32-audioI2S/assets/65090450/3b28c27f-8a2f-481d-9a97-fe7589433856)

**Comment by Finnn-glitch** [comment]
Also do i need PSRAM in order to work?
I bought an ESP32 S3 dev kit especially for that purpose.
Also tried with PSRAM enabled still nothing.

**Comment by schreibfaul1** [comment]
It is possible that the described error occurs in connection with Arduino version 2.0.12. I was made aware of this with a pull request and the error is now eliminated. And V2.0.12 has only been around for a week. Basically, the library works without PSRAM. Some memory are needed for buffering the audio data, which are relocated to the PSRAM if possible.

**Reply by Finnn-glitch** [reply]
Hm weiered im on Arduino version 1.8.19 because i hate the new design xD.
I got my old esp32 with no psram and 4kb and it works fine.
